### DISCUSSION TITLE
How to Add color if the note contains a particular Heading?

### DISCUSSION THREAD

**Discussion by ettezo** [marcusolsson/obsidian-projects#697] [Help] [answered]
Im trying to give color to cards whose contain notes contain a particular H1 heading.

My setup is like this: 
![image](https://github.com/marcusolsson/obsidian-projects/assets/85111678/7378df9e-fa25-4023-a813-934328ef3e6e)

and no card gets the color despite the notes having that particular heading. 

Is my syntax wrong? Or does the plugin not recognize Headings or is there any other way to implement this?
Also, can we use any frontmatter for this purpose?

Any help is deeply appreciated.

**Comment by Acylation** [comment]
Hello, can I assume that you are using dataview as the source? Other sources only gathering information from file properties (YAML frontmatters), and only dataview have the chance to touch some contents in the body text (inline field, tasks etc.)

Please note that the column `File` here refers to a link to the file with the file name as display text. Thus, accessing body text from the `File` is impossible. As I don't personally use dataview, I performed a quick search on google and failed to access headings via dataview queries.

In summary, it seems there's currently no way to access headings via Projects.

**Reply by ettezo** [reply, marked as answer]
Thanks for the reply and yes I'm using dataview. Also I found a workaround after some tinkering, I changed from using heading to using frontmatter, and for Project to use a particular frontmatter in Table or Board view you have to query it via dataview.

So in my case, I was trying to give different colors to notes according to their type(whether they were 'story' or 'task') so instead of searching for a particular heading(which would indicate the type of note) I added 'type' as a frontmatter to the notes and queried it in dataview. So now project is able to access it in both the views and I'm able to apply different colors according to the type.

![image](https://github.com/marcusolsson/obsidian-projects/assets/85111678/c08dc9bf-927a-4a6c-82d9-6e7c6ae7212d)
